Changelog.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## December 16, 2020 <Badge text="beta" type="success" />
+
+Released on December 16, 2020.
+
+### Enhancements
+
+- Add the `events` RBAC permission to agents created by the helm chart - [#157](https://github.com/PrefectHQ/cloud/pull/157)
+
```

Comment:
How'd this get in here?

Reply:
I accidentally cut the changelog for [a release from that day](https://github.com/PrefectHQ/server/releases/tag/2020-12-16) locally, but never pushed up my changes 😬